packages/integration-karma/test/shadow-dom/ShadowRoot.elementsFromPoint/index.spec.js

```diff
@@ -0,0 +1,131 @@
+            const elm = createElement('x-container', { is: Container });
+            document.body.appendChild(elm);
+
+            const { body } = document;
+            const html = document.documentElement;
+            const h1 = elm.shadowRoot.querySelector('h1');
+            const inContainer = elm.shadowRoot.querySelector('.in-container');
+            const slottable = elm.shadowRoot.querySelector('x-slottable');
+            const slotted = elm.shadowRoot.querySelector('.slotted');
+
+            test(elm, [elm, body, html]);
+            test(h1, [h1, elm, body, html]);
+            test(inContainer, [slotted, slottable, inContainer, elm, body, html]);
+            test(slottable, [slotted, slottable, inContainer, elm, body, html]);
+            test(slotted, [slotted, slottable, inContainer, elm, body, html]);
+        });
+    }
+
+    if (!isIE && !isSafari && !isOldChrome) {
+        // Safari and old Chrome return unpredictable results
+        it('advanced shadow example', () => {
+            const elm = createElement('x-container', { is: Container });
+            document.body.appendChild(elm);
+
+            const { body } = document;
+            const html = document.documentElement;
+            const inContainer = elm.shadowRoot.querySelector('.in-container');
+            const slottable = elm.shadowRoot.querySelector('x-slottable');
+            const component = elm.shadowRoot.querySelector('x-component');
+            const inComponent = component.shadowRoot.querySelector('.in-component');
+            const inComponentInner = component.shadowRoot.querySelector('.in-component-inner');
+            const slotWrapper = slottable.shadowRoot.querySelector('.slot-wrapper');
+            const inSlottable = slottable.shadowRoot.querySelector('.in-slottable');
+            const inSlottableInner = slottable.shadowRoot.querySelector('.in-slottable-inner');
+
+            test(component, [component, slottable, inContainer, elm, body, html]);
+            test(inComponent, [
+                inComponentInner,
+                inComponent,
+                component,
+                slottable,
+                inContainer,
+                elm,
+                body,
+                html,
+            ]);
+            test(inComponentInner, [
+                inComponentInner,
+                inComponent,
+                component,
+                slottable,
+                inContainer,
+                elm,
+                body,
+                html,
+            ]);
+            test(slotWrapper, [slotWrapper, slottable, inContainer, elm, body, html]);
+            test(inSlottable, [inSlottable, slottable, inContainer, elm, body, html]);
+            test(inSlottableInner, [
+                inSlottableInner,
+                inSlottable,
+                slottable,
+                inContainer,
+                elm,
+                body,
+                html,
+            ]);
+        });
+    }
+});
```

Comment:
I tried adding a WebDriver integration test as well (to reproduce Chromedriver calling `elementsFromPoint` [here](https://github.com/bayandin/chromedriver/commit/ad6ede81b0d048e031fbe7cc877dcb7d7b1f0049)), but for the life of me, I just couldn't figure out how to actually repro this.

It seems to only occur with Chromedriver running in some kind fo special state. I tried various modes (`w3c: false|true`, various WebDriver APIs (getLocation, moveTo), and Chromedriver vs Selenium), but could not repro.